src/devlog.erl

```diff
@@ -1,31 +1,13 @@
 -module(devlog).
 
--export([
-    log/1,
-    log_inner/1
-]).
+-export([log/1]).
 
 -define(DEVLOG, server_devlog).
 
--doc """
-If `devlog` is not running on our current node, broadcast the log message to all
-connected nodes once.
-""".
 log(Msg) ->
-    case erlang:whereis(?DEVLOG) of
-        undefined ->
-            erpc:multicast(nodes(), ?MODULE, log_inner, [Msg]);
-        _Pid ->
-            gen_server:cast(?DEVLOG, {log, Msg})
-    end.
-
--doc """
-DO NOT CALL THIS FUNCTION DIRECTLY
-""".
-log_inner(Msg) ->
-    case erlang:whereis(?DEVLOG) of
+    case global:whereis_name(?DEVLOG) of
```

Comment:
Starting `server_devlog` with `{global, ?MODULE}` registers the name globally. So, my interface functions can just look for it and then call/cast directly.